exercises/practice/collatz-conjecture/tst/collatz-conjecture/collatz-conjecture.cut

```diff
@@ -0,0 +1,24 @@
+TestCase "zero steps for one"
```

Comment:
@KTSnowy,
This would make the tests pass, but assumes that the counter is always called `WS-STEPS`.
```suggestion
TESTSUITE 'Tests for collatz-conjecture'

BEFORE-EACH
	MOVE 0 TO WS-STEPS
END-BEFORE

TestCase "zero steps for one"
```

Reply:
The WS-STEPS counter is included in the starting source file, so all students will use that one to count the steps (I think). So I don't think that will be an issue.